scipy/interpolate/_bsplines.py

```diff
@@ -314,6 +322,14 @@ def __call__(self, x, nu=0, extrapolate=None):
         x = np.asarray(x)
         x_shape, x_ndim = x.shape, x.ndim
         x = np.ascontiguousarray(x.ravel(), dtype=np.float_)
+
+        # With periodic extrapolation we map x to the segment
+        # (self.t[k], self.t[-k]).
+        if extrapolate == 'periodic':
```

Comment:
If I am not wrong, `(self.t[k], self.t[-k])` refers to open interval, hence both `t[k]` and `t[-k]` are excluded.